### PR TITLE
=doc #18917 remove old API mentions from the docs

### PR DESCRIPTION
**Pull request by johanandren**
The places I found, there might be more, but want to PR this before @rkuhn starts renaming and I get a merge conflict to work on for the rest of the day.

Refs #18917

### CODE REVIEW

File: akka-docs-dev/rst/java/stream-quickstart.rst [hunk 1]
```diff
@@ -103,27 +103,28 @@ Elements that can be used to form such "fan-out" (or "fan-in") structures are re
...
 
-A graph can be either ``closed`` which is also known as a "*fully connected graph*", or ``partial`` which can be seen as
-a *partial graph* (a graph with some unconnected ports), thus being a generalisation of the Flow concept, where ``Flow``
-is simply a partial graph with one unconnected input and one unconnected output. Concepts around composing and nesting
-graphs in large structures are explained explained in detail in :ref:`composition-java`.
+.. includecode:: ../../../akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/TwitterStreamQuickstartDocTest.java#flow-graph-broadcast
 
-It is also possible to wrap complex computation graphs as Flows, Sinks or Sources, which will be explained in
-detail in :ref:`constructing-sources-sinks-flows-from-partial-graphs-java`. FlowGraphs are constructed like this:
+As you can see, we use graph builder ``b`` to construct the graph using ``UniformFanOutShape`` and ``Flow``s.
```
Comment (ktoso) [inline akka-docs-dev/rst/java/stream-quickstart.rst]: This breaks the output:

```
``Flow``s
```

must be:

```
``Flow`` s
```

File: akka-docs-dev/rst/scala/stream-quickstart.rst [hunk 2]
```diff
@@ -99,32 +99,31 @@ Elements that can be used to form such "fan-out" (or "fan-in") structures are re
...
-FlowGraphs are constructed like this:
+Graphs are constructed using :class:`FlowGraph` like this:
 
 .. includecode:: code/docs/stream/TwitterStreamQuickstartDocSpec.scala#flow-graph-broadcast
 
-.. note::
-  The ``~>`` (read as "edge", "via" or "to") operator is only available if ``FlowGraph.Implicits._`` are imported.
-  Without this import you can still construct graphs using the ``builder.addEdge(from,[through,]to)`` method.
+As you can see, inside the :class:`FlowGraph` we use an implicit graph builder ``b`` to mutably construct the graph
+using the ``~>`` "edge operator" (also read as "connect" or "via" or "to"). The operator is provided implicitly
```
Comment (ktoso) [inline akka-docs-dev/rst/scala/stream-quickstart.rst]: oh, not "squiggly arrow"? // joking :wink:

**Comment by ktoso** [comment]
One formatting fix and it's LGTM, thanks! Please squash so I can merge soon

**Comment by drewhk** [comment]
LGTM. Don't wait for validation

**Comment by ktoso** [comment]
Fixed the small typo and merged as https://github.com/akka/akka/commit/922f51b0d5db886b7243e1f255e77c24defd2bb9

**Comment by drewhk** [comment]
Conflicts

**Comment by drewhk** [comment]
wot.

**Comment by ktoso** [comment]
I fixed the typo I spotted and merged it using the command line, same result and author perserved etc.

**Pull request closed without merging**